components/mediation/extensions/pom.xml

```diff
@@ -33,6 +34,7 @@
 
     <modules>
         <module>org.wso2.micro.integrator.transport.handlers</module>
+        <module>UTSecurity</module>
```

Comment:
isn't it the handler we brougt in? In that case, can we move this inside handler? Also let's follow the same naming convention when naming modules

Reply:
fixed